erpnext/projects/doctype/task/task_list.js

```diff
@@ -25,7 +25,9 @@ frappe.listview_settings['Task'] = {
 		return [__(doc.status), colors[doc.status], "status,=," + doc.status];
 	},
 	gantt_custom_popup_html: function(ganttobj, task) {
-		var html = `<h5>${ganttobj.name}</h5>`;
+		var html = `<h5><a style="text-decoration:underline"\
```

Comment:
This will also work I think: `<a href="#Form/Task/${ganttobj.id}">`

Reply:
Yep, it does. Updated!